### PR TITLE
creating json

### PR DESCRIPTION
**Pull request by muskan9650**

### CODE REVIEW

File: js-if-else_json/if-Statement.json [hunk 1]
```diff
@@ -0,0 +1,54 @@
+[
+    {
+        "component": "Code block",
```
Comment (pysaquib) [inline js-if-else_json/if-Statement.json:3]: Replace "Code Block" with just "code"

File: js-if-else_json/if-Statement.json [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        "type": "Javascript",
```
Comment (pysaquib) [inline js-if-else_json/if-Statement.json:4]: Please understand that "Javascript" != "javascript". Things like this should not be in title case. Make it in all small

File: js-if-else_json/if-Statement.json [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+        "Text": "if (condition is true) {statement is executed}"
```
Comment (pysaquib) [inline js-if-else_json/if-Statement.json:5]: Where in the sample json file you found `Text` key instead of `text` key

Comment (pysaquib) [reply js-if-else_json/if-Statement.json:5]: Please make it `text`. Keys are important. Client would never be able to find `something.Text`

File: js-if-else_json/if-Statement.json [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+    },
+    {
+        "component": "Block quote",
```
Comment (pysaquib) [inline js-if-else_json/if-Statement.json:8]: no space in between and all small

File: js-if-else_json/if-Statement.json [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+        "value": " If you want to run a piece of code, based on a certain condition, then you should use the if statement.The keyword if tells JavaScript to execute the code in the curly braces under certain conditions, defined in the parentheses ().\n"
+    },
+    {
+        "component": "text", 
+        "value": "**Note**",
```
Comment (pysaquib) [inline js-if-else_json/if-Statement.json:13]: If you already defined `bold` in decoration. No need to give ** ** in value. Just give the text

File: js-if-else_json/if-Statement.json [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+        "decoration":{
+          "type":"bold"
+        }
+    },
+    {
+        "component": "text",
+        "value": ": These conditions are known as Boolean conditions and they are only be true or false.\n"
```
Comment (pysaquib) [inline js-if-else_json/if-Statement.json:20]: Where is the opening "?

File: js-if-else_json/if-Statement.json [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+    },
+    {
+        "component": "Block quote",
```
Comment (pysaquib) [inline js-if-else_json/if-Statement.json:23]: blockquote

File: js-if-else_json/if-Statement.json [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+        "value": " When the condition evaluates to true, the program executes the statement inside the curly braces. When the Boolean condition evaluates to false, the statement inside the curly braces will not execute.\n"
+    },
+    {
+        "component": "Block quote",
```
Comment (pysaquib) [inline js-if-else_json/if-Statement.json:27]: blockquote

File: js-if-else_json/if-Statement.json [hunk 9]
```diff
@@ -0,0 +1,54 @@
...
+        "value": "Let us see some more examples."
+    },
+    {
+        "component": "Text ", 
+        "value": "**Example**",
```
Comment (pysaquib) [inline js-if-else_json/if-Statement.json:32]: remove **

File: js-if-else_json/if-Statement.json [hunk 10]
```diff
@@ -0,0 +1,54 @@
...
+        "decoration":{
+          "type":"bold"
+        }
+    },
+    {
+        "component": "Code block ",
```
Comment (pysaquib) [inline js-if-else_json/if-Statement.json:38]: code

File: js-if-else_json/if-Statement.json [hunk 11]
```diff
@@ -0,0 +1,54 @@
...
+        "type": "Javascript",
```
Comment (pysaquib) [inline js-if-else_json/if-Statement.json:39]: javascript

File: js-if-else_json/if-Statement.json [hunk 12]
```diff
@@ -0,0 +1,54 @@
...
+        "Text": "var a = 10; var b = 12; if(b > a){ console.log( b,' is greater than  ', a  );}"
```
Comment (pysaquib) [inline js-if-else_json/if-Statement.json:40]: text

File: js-if-else_json/if-Statement.json (hunk 9 above)
Comment (pysaquib) [reply js-if-else_json/if-Statement.json:32]: Also after Example there was `:-`
Make a new json obj for it. type will be text

File: js-if-else_json/if-Statement.json [hunk 13]
```diff
@@ -0,0 +1,54 @@
...
+    },
+    {
+        "component": "text",
+        "value": "Output :- 12, is greater than, 10"
+    },
+    {
+        "component": "H3",
```
Comment (pysaquib) [inline js-if-else_json/if-Statement.json:47]: component is `header`
variant is `3`
Check the sample file `js-intro_json` for reference

File: js-if-else_json/if-Statement.json [hunk 14]
```diff
@@ -0,0 +1,54 @@
...
+        "value": "**Explanation**:"
+    },
+    {
+        "component": "text",
+        "value": "    Here 12 is greater than 10. So, the console will show us the output as 12."
+    }
+]
```
Comment (pysaquib) [inline js-if-else_json/if-Statement.json:54]: Press enter after this line

**Review by pysaquib — CHANGES_REQUESTED** [review]

File: js-if-else_json/if-Statement.json (hunk 6 above)
Comment (muskan9650) [reply js-if-else_json/if-Statement.json:20]: the opening inverted comma is between the colons :  ":

File: js-if-else_json/conditional-statements.json [hunk 15]
```diff
@@ -0,0 +1,33 @@
+[
+    {
+        "component": "blockquote",
+        "value": "Consider the following:\n"
+    },
+    {
+        "component": "code ",
```
Comment (pysaquib) [inline js-if-else_json/conditional-statements.json:7]: extra space after `code`

File: js-if-else_json/else-Statement.json [hunk 16]
```diff
@@ -0,0 +1,61 @@
...
+    },
+    {
+        "component": "text", 
+        "value": "Example\n",
+        "decoration":{
+          "type":"bold"
+        }
+    },
+    {
+        "component": "code ",
```
Comment (pysaquib) [inline js-if-else_json/else-Statement.json:14]: space

**Review by pysaquib — CHANGES_REQUESTED** [review]
fix

File: js-if-else_json/index.md [hunk 17]
```diff
@@ -0,0 +1,12 @@
+- conditional-statements.md
```
Comment (pysaquib) [inline js-if-else_json/index.md:1]: Will it be `.md`?

**Review by pysaquib — CHANGES_REQUESTED** [review]
fix this

**Pull request merged**